src/convertToObject.js

```diff
@@ -6,7 +6,30 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const array = sourceString
+    .split(';')
+    .map((style) => style.trim())
+    .filter((style) => style.length > 0);
```

Comment:
[CODE STYLE] - Avoid variable namings like `array`. It will decrease code readability. Consider naming the variable in a way that reflects its content or purpose, such as `styleDeclarations` or `styleEntries`.